syntax = "proto3";

package rcl;

service RLC {

    rpc GetAction(ObservationData) returns (ActionData) {}

}

message ObservationData {
    repeated float features = 1;
    float reward = 2;
    int32 done = 3; 
}

/*
message Decision {
    int32 action_index = 1;
    bool is_end = 2;
}
*/

message ActionData {
    // Decision rl_return = 1;
    int64 action_index = 1;
    bool is_end = 2;
}